src/ledger/LedgerManagerImpl.cpp

```diff
@@ -993,22 +1000,22 @@ LedgerManagerImpl::closeLedger(LedgerCloseData const& ledgerData,
     auto ledgerSeq = ltx.loadHeader().current().ledgerSeq;
     if (protocolVersionStartsFrom(maybeNewVersion, SOROBAN_PROTOCOL_VERSION))
     {
-        updateNetworkConfig(ltx);
+        updateSorobanNetworkConfigForApply(ltx);
     }
 
-    auto closeLedgerResult =
-        ledgerClosed(ltx, ledgerCloseMeta, initialLedgerVers);
+    LedgerState appliedLedgerState = sealLedgerTxnAndStoreInBucketsAndDB(
+        ltx, ledgerCloseMeta, initialLedgerVers);
 
     if (ledgerData.getExpectedHash() &&
-        *ledgerData.getExpectedHash() != closeLedgerResult.ledgerHeader.hash)
+        *ledgerData.getExpectedHash() != appliedLedgerState.ledgerHeader.hash)
     {
         throw std::runtime_error("Local node's ledger corrupted during close");
     }
 
     if (mMetaStream || mMetaDebugStream)
     {
         releaseAssert(ledgerCloseMeta);
-        ledgerCloseMeta->ledgerHeader() = closeLedgerResult.ledgerHeader;
+        ledgerCloseMeta->ledgerHeader() = appliedLedgerState.ledgerHeader;
```

Comment:
Nit: Maybe `ledgerCloseMeta` -> `ledgerApplyMeta`. I know the XDR struct is called `ledgerCloseMeta` itself, but this is really the meta from the current apply ledger, not the LCL ledger, which this kinda sounds like at the moment. But maybe it's more confusing to not align with the XDR struct name here, I'm not sure

Reply:
yeah I considered it but as with LedgerCloseData I decided this terminology has bled too far outside of LM to maintain the apply-vs-close distinction there. I'd like to leave this as is.